.github/workflows/enable-auto-merge.yml

```diff
@@ -0,0 +1,29 @@
+name: Enable auto merge
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  add_milestone:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'dotnet/roslyn-analyzers' && github.event.pull_request.author.login == 'dotnet-bot' && startsWith(github.event.pull_request.title, 'Localized file check-in') }}
```

Comment:
The title will be something like "Localized file check-in by OneLocBuild Task: Build definition ID 830: Build ID 1208815". So we would want some way to say `if it starts with 'Localized file check-in'`

Reply:
I think you need to scroll right 😉

Reply:
ah I see it :)